lib/matplotlib/backends/_backend_tk.py

```diff
@@ -507,16 +507,23 @@ class NavigationToolbar2Tk(NavigationToolbar2, tk.Frame):
     Attributes
     ----------
     canvas : `FigureCanvas`
-        the figure canvas on which to operate
+        The figure canvas on which to operate.
     win : tk.Window
-        the tk.Window which owns this toolbar
-
+        The tk.Window which owns this toolbar.
+    pack_toolbar : bool, optional
+        If True, add the toolbar to the parent's pack manager's packing list
+        during initialization with *side='bottom'* and *fill='x'*.
+        If you want to use the toolbar with a different layout manager, you
+        must first call *pack_forget* on the toolbar object.
```

Comment:
I was confused by this sentence when first reading it, After all, the change proposed here is meant to prevent the need to call `pack_forget`. Maybe move it to the bottom and let it say

> If you want to use the toolbar with a different layout manager, use ``pack_toolbar=False``.